storage/badger/operation/common.go

```diff
@@ -14,7 +14,37 @@ import (
 	"github.com/onflow/flow-go/storage"
 )
 
-// insert will encode the given entity using JSON and will insert the resulting
+// batchInsert will encode the given entity using msgpack and will upsert the resulting
+// binary data in the badger wrote batch under the provided key - if the value already exists
+// in the database it will be overridden
```

Comment:
so it's more of a batchUpsert then, i guess? do we foresee any problems or is this behaviour change not likely to have a big impact

Reply:
It is, I remember for some reason we didn't like this name in the past, however.